contracts/example/tests/test.rs

```diff
@@ -329,6 +324,9 @@ fn its_example() {
     ]
     .join("\n\n"));
 
-    let token = token::TokenClient::new(&env, &test_config.its_client.token_address(&token_id));
-    assert_eq!(token.balance(&test_config.app.address), 0);
+    let token = token::TokenClient::new(&env, &its_client.token_address(&token_id));
+    assert_eq!(token.balance(&example_app.address), 0);
+
+    let recipient_address = Address::from_string_bytes(&data.unwrap());
+    assert_eq!(token.balance(&recipient_address), amount);
```

Comment:
```suggestion
    let recipient = Address::from_string_bytes(&data.unwrap());
    assert_eq!(token.balance(&recipient), amount);
```
no need to suffix variables with their type